lambda-notes/src/components/Search.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { search_results_clicked } from '../actions/index';
+
+import './css/Search.css';
+
+class Search extends React.Component {
+  state = {
+    title: '',
+    body: '',
+  };
+
+  render() {
+    return (
+      <div className="search">
+        <h1 className="search-header">Search for a Title</h1>
+        <input className="searchbar" onChange={this.handleTitleChange} />
+        <h1 className="search-header"> Or... </h1>
+        <h1 className="search-header">Search within the body</h1>
+        <input className="searchbar" onChange={this.handleBodyChange} />
+        <button className="search-button" onClick={this.handleSearch}>
+          Search
+        </button>
+      </div>
+    );
+  }
+  handleTitleChange = event => {
+    this.setState({ title: event.target.value });
+  };
+  handleBodyChange = event => {
+    this.setState({ body: event.target.value });
+  };
+  handleSearch = () => {
+    let results = [];
+    if (this.state.title === '') {
```

Comment:
Another great stretch goal functionality, great idea for a Search functionality, amazing.